custom/icds_reports/utils/aggregation_helpers/distributed/agg_awc.py

```diff
@@ -283,24 +290,29 @@ def updates(self):
                     ucr.supervisor_id = adolescent_girls_table.supervisor_id AND
                     adolescent_girls_table.month=%(start_date)s
                     )
+                 LEFT JOIN
+                 "{migration_table}" agg_migration ON (
+                    ucr.doc_id = agg_migration.person_case_id AND
+                    agg_migration.month = %(start_date)s AND
+                    ucr.supervisor_id = agg_migration.supervisor_id
+                 )
             WHERE (opened_on <= %(end_date)s AND
               (closed_on IS NULL OR closed_on >= %(start_date)s )) AND
-              migration_status IS DISTINCT FROM 1
+              ((agg_migration.is_migrated=1 AND
```

Comment:
same comment about this logic applies everywhere

Reply:
done!